cmd/tailscale/cli/funnel.go

```diff
@@ -20,7 +20,13 @@ import (
 	"tailscale.com/util/mak"
 )
 
-var funnelCmd = newFunnelCommand(&serveEnv{lc: &localClient})
+var funnelCmd = func() *ffcli.Command {
+	se := &serveEnv{lc: &localClient}
+	if os.Getenv("TAILSCALE_FUNNEL_V2") == "on" {
```

Comment:
Nit: TAILSCALE_FUNNEL_DEV

A little clearer that this is still in development

Reply:
Let's rename the associated files/methods to match if we do that.